app/src/main/res/values/strings.xml

```diff
@@ -29,6 +29,10 @@
     <string name="sign_in_to_add_favorites">Please sign in to add favorites</string>
     <string name="filter">Filter</string>
     <string name="close">Close</string>
+    <string name="delete_dialog_title">Delete Item</string>
```

Comment:
They are the same as the ones below that you had already added. I think you misunderstood me  when i said that the string resources were missing. The messages were already in strings.xml but you just forgot to use them in the ReportScreenMolecules.

Reply:
I didn't, if you see they're not quite the same. I decided to replace "Image" with "Item" to keep the molecule generic

Reply:
My point was that some were the exact same. All good now, whichever you kept doesnt bother me really